exercises/concept/weather-report/report-results

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+db_file=$1
+mapfile -t slugs < <(jq -r 'keys[]' test_data.json)
```

Comment:
FWIW it's worth noting that this will fail on Mac with the default/older `bash`.

Reply:
Any Linux system ought to have bash5 😄  I don't think the exact message matters much, thought for Mac users (the primary target of that message) a nudge towards Homebrew could be handy.

Reply:
Since we have installation instructions for Windows, it's a bad look if the first concept exercise can't run on Windows.

I fired up a Windows VM and installed sqlite (via chocolatey: did you want a PR with those instructions?) -- the sqlite3 cli's `.shell` command uses `cmd`, so we'd want a batch file equivalent of the shell script.

Anybody want to take that on?

Reply:
I might give it a try, if I can spin up a Windows VM somehow without buying a license.